compiler/damlc/daml-doc/src/DA/Daml/Doc/Extract.hs

```diff
@@ -239,11 +239,11 @@ getFctDocs ctx@DocCtx{..} (DeclData decl docs) = do
         fct_descr = docs
 
     guard (exportsFunction dc_exports fct_name)
-    guard (not $ "_choice_" `T.isPrefixOf` packRdrName name)
-    guard (not $ "_interface_instance_" `T.isPrefixOf` packRdrName name)
-    guard (not $ "_requires_" `T.isPrefixOf` packRdrName name)
-    guard (not $ "_method_" `T.isPrefixOf` packRdrName name)
-    guard (not $ "_view_" `T.isPrefixOf` packRdrName name)
+    guard (not $ "_choice$_" `T.isPrefixOf` packRdrName name)
+    guard (not $ "_interface_instance$_" `T.isPrefixOf` packRdrName name)
+    guard (not $ "_requires$_" `T.isPrefixOf` packRdrName name)
+    guard (not $ "_method$_" `T.isPrefixOf` packRdrName name)
+    guard (not $ "_view$_" `T.isPrefixOf` packRdrName name)
```

Comment:
This module already imports stuff from `ghc-lib{,-parser}`, so I think we could use something like `guard (not . isDamlGenerated . rdrNameOcc $ name)` here

Reply:
fixed